Shared/Samples/Set visibility of subtype sublayer/SetVisibilityOfSubtypeSublayerView.Model.swift

```diff
@@ -0,0 +1,164 @@
+            labelDefinition.placement = .pointAboveRight
+            labelDefinition.usesCodedValues = true
+            return labelDefinition
+        }()
+        
+        // The formatter used to generate strings from scale values.
+        private let scaleFormatter: NumberFormatter = {
+            let numberFormatter = NumberFormatter()
+            numberFormatter.numberStyle = .decimal
+            numberFormatter.maximumFractionDigits = 0
+            return numberFormatter
+        }()
+        
+        /// A Boolean value indicating whether the settings should be presented.
+        @Published var isShowingSettings = false
+        
+        /// A Boolean value indicating whether to show the subtype sublayer.
+        @Published var showsSublayer = true
+        
+        /// A Boolean value indicating whether to show the subtype sublayer's renderer.
+        @Published var showsOriginalRenderer = true
+        
+        /// The current scale of the map.
+        var currentScale: Double = .zero
+        
+        /// The map's current scale value in text.
+        @Published var currentScaleText: String = ""
+        
+        /// The subtype sublayer's minimum scale value in text.
+        @Published private(set) var minimumScaleText: String = "Not Set"
+        
+        init() {
+            map.initialViewpoint = .initialViewpoint
+            subtypeFeatureLayer = SubtypeFeatureLayer(featureTable: featureTable)
+            subtypeFeatureLayer.scalesSymbols = false
+        }
+        
+        deinit {
+            ArcGISEnvironment.authenticationManager.arcGISCredentialStore.removeAll()
+        }
+        
+        /// Performs important tasks including adding credentials, loading and adding operational layers.
+        @MainActor
+        func setup() async throws {
+            try await ArcGISEnvironment.authenticationManager.arcGISCredentialStore.add(.publicSample)
+            try await subtypeFeatureLayer.load()
+            map.addOperationalLayer(subtypeFeatureLayer)
+            subtypeSublayer = subtypeFeatureLayer.sublayer(withSubtypeName: "Street Light")
+            subtypeSublayer?.labelsAreEnabled = true
+            originalRenderer = subtypeSublayer?.renderer
+            subtypeSublayer?.addLabelDefinition(labelDefinition)
+        }
+        
+        func toggleSublayer() {
+            subtypeSublayer?.isVisible = showsSublayer
+        }
+        
+        func toggleRenderer() {
+            if showsOriginalRenderer {
+                subtypeSublayer?.renderer = originalRenderer
+            } else {
+                let symbol = SimpleMarkerSymbol(style: .diamond, color: .systemPink, size: 20)
+                let alternativeRenderer = SimpleRenderer(symbol: symbol)
+                subtypeSublayer?.renderer = alternativeRenderer
+            }
+        }
+        
+        func formatCurrentScaleText() {
+            currentScaleText = "1:\(currentScale.formatted(.decimal))"
```

Comment:
no need to change, another way can be `1:\(currentScale, specifier: "%.0f")`, although it has a comma separator.